framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -100,12 +100,15 @@ public void run() {
       query = queries[mainQueryIndex];
       executeQuery(query);
       
-      testVerifier = new TestVerifier(columnTypes, query, columnLabels, matrix.verificationTypes);
+      testVerifier = new TestVerifier(columnTypes, query, columnLabels, matrix.verificationTypes, modeler.categories);
       if (query.startsWith("explain") || matrix.verificationTypes.get(0).equalsIgnoreCase("regex") ||
           matrix.verificationTypes.get(0).equalsIgnoreCase("regex-no-order") ||
           matrix.verificationTypes.get(0).equalsIgnoreCase("filter-ratio")) {
         setTestStatus(testVerifier.verifyTextPlan(modeler.expectedFilename, outputFilename));
+      } else if (matrix.verificationTypes.get(0).equalsIgnoreCase("text") ) {
+        setTestStatus(testVerifier.verifyText(modeler.expectedFilename, outputFilename));
```

Comment:
It is a new verification type.  I was originally going to use it for the JDBC tests, but I don't need it for that any more.  I am now planning to use it for orderby testing.